scripts/migrate.js

```diff
@@ -0,0 +1,46 @@
+const { migrate } = require('postgres-migrations')
+const path = require('path')
+
+const { NODE_ENV } = process.env
+
+// This code is repeated a few times in our scripts. Can you factor it out into a shared helper function?
```

Comment:
This should be removed. If you want to attempt this at some point and are trying to remind yourself of it the best place to make a note of it would be in the README